march_hardware/include/march_hardware/Slave.h

```diff
@@ -30,8 +30,9 @@ class Slave
 
   virtual ~Slave() noexcept = default;
 
-  virtual void writeInitialSDOs(int /* cycle_time */)
+  virtual bool writeInitialSDOs(int /* cycle_time */)
```

Comment:
Perhaps it would be nice to document the expected return value here? When is it `true` and when is it `false`? You can do so in docstring
```cpp
/**
 * Writes initial SDOs.
 *
 * @return True when ..., false when...
 */

Reply:
I will write a short docstring about this function